DevEdu.API/DevEdu.DAL/Repositories/CommentRepository.cs

```diff
@@ -110,6 +110,9 @@ public int UpdateComment(CommentDto commentDto)
                 },
                 commandType: CommandType.StoredProcedure
             );
+
+            var commentId = commentDto.Id;
+            return GetComment(commentId);
```

Comment:
it's not correct to have multiple calls to DB in one repo method
please do it in BLL